src/components/create-user.html

```diff
@@ -66,8 +76,26 @@ <h1>Create User</h1>
         }
       }
 
-      subBut(){
-        this.storage.createUser(this.newUserData);
+      async getAndAddImgUrl() {
+        const name = this.newUserData.firstName;
+        let imageUrl = '';
+        await profileImage.findImages(name).then(results => imageUrl = results.primaryImage);
```

Comment:
```suggestion
        const results = await profileImage.findImages(name);
        imageUrl = results.primaryImage;
```
or just
```suggestion
       profileImage.findImages(name).then(results => imageUrl = results.primaryImage);
```
Mixing async with `.then` is a bit unnecessary as `async` generally replaces the need for `.then`.

Reply:
I agree it does seem like there should be a simpler way to do this! The first suggestion sort of works, it returns images but also returns undefined consistently so something is broken because the other way it doesn't do that, it also takes longer to get results -  the second suggestion breaks everything - I've tried various ways of doing it without await and .then and haven't been able to make it work. I think it needs to be await because it just skips over everything because the images don't come in fast enough for there to be something if it's not async- and my understanding about .then is that you need it if you want the results of the promise - If i just try to access the await results it console logs promise pending - and when i researched the issue the examples i found used .then to get results of an await.